src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUntypedObject.cs

```diff
@@ -52,6 +52,111 @@ public static FormulaValue Index_UO(IRContext irContext, FormulaValue[] args)
             }
         }
 
+        public static FormulaValue First_UO(IRContext irContext, UntypedObjectValue[] args)
+        {
+            var arg0 = (UntypedObjectValue)args[0];
+            var element = arg0.Impl;
+            var len = element.GetArrayLength();
+
+            if (len == 0)
+            {
+                return new BlankValue(irContext);
+            }
+
+            var result = element[0];
+
+            // Map null to blank
+            if (result == null || result.Type == FormulaType.Blank)
```

Comment:
>result == null ||  [](http://example.com/codeflow?start=16&length=18)

Can this ever be null? From what I see, something like `Index(ParseJSON("[null,2,3]"),1)` is an instance of JsonUntypedObject with Type = FormulaType.Blank

Reply:
Removed